.github/workflows/schema_spy.yml

```diff
@@ -0,0 +1,138 @@
+name: "Build & Publish DB Documentation"
+
+on:
+  push:
+    branches:
+      #- main
+      #- production
```

Comment:
These will replace the following line before going into production. This allows for this to be tested by pushing on this branch.